core/src/components/toggle/toggle.tsx

```diff
@@ -417,6 +413,8 @@ export class Toggle implements ComponentInterface {
         aria-disabled={disabled ? 'true' : null}
         tabindex={disabled ? undefined : 0}
         onKeyDown={this.onKeyDown}
+        onFocus={() => this.onFocus()}
+        onBlur={() => this.onBlur()}
```

Comment:
```suggestion
        onFocus={this.onFocus}
        onBlur={this.onBlur}
```

Are you able to do this instead? When you do it the way you currently you, you're creating a new closure on every render. It's not a *huge* issue, but it should be unnecessary and we generally avoid it where possible.